website1/src/scss/sections/_main-items-bunker.scss

```
@@ -0,0 +1,135 @@
+.main-items-bunker {
+  &-wrapper {
+    width: 100%;
+    background-color: $grey-light;
+  }
+
+  &__item-box {
+    margin: 71px 49px 220px 110px;
+    position: relative;
+  }
+
+  &__item {
+    position: absolute;
+
+    &-title {
+      text-transform: uppercase;
+      font-size: 24px;
+      line-height: 30px;
+      margin-left: 45px;
+    }
+
+    &-text {
+      font-weight: 300;
+      line-height: 22px;
+      margin-top: 5px;
+      margin-left: 45px;
+
+      &-overlap {
+        max-width: 326px;
+      }
+
+      &-door {
+        max-width: 475px;
+      }
+
+      &-hatch {
+        max-width: 360px;
+      }
+
+      &-ventilation {
+        max-width: 306px;
+      }
+    }
+
+    &-door {
+      bottom: -120px;
+    }
```

Add pointer styles here.
```suggestion
    &-door {
      bottom: -120px;
      &-pointer {
        bottom: 125px;
      }
    }
```

---

The same for other items.